terraform/install.sh

```diff
@@ -59,6 +57,45 @@ loadGen() {
   ../loadgenerator-tool startup --zone us-west2-a $external_ip
 }
 
+displaySuccessMessage() {
+    created_project=$(cat ./terraform.tfstate | \
+                        grep "\"project\":" | \
+                        grep -oh "stackdriver-sandbox-[0-9]*" | \
+                        head -n 1)
+
+    gcp_path="https://console.cloud.google.com/kubernetes/workload"
+    if [[ -n "${created_project}" ]]; then
+        gcp_path="$gcp_path?project=$created_project"
+    fi
+    TRIES=0
+    while [[ -z "${loadgen_ip}" && "${TRIES}" -lt 20  ]]; do
+        sleep 1
+        loadgen_ip=$(gcloud compute instances list --project "$created_project" \
+                                                   --filter="name:loadgenerator*" \
+                                                   --format="value(networkInterfaces[0].accessConfigs.natIP)")
+        TRIES=$((TRIES + 1))
+    done
+
+    if [[ -n "${loadgen_ip}" ]]; then
+        loadgen_ip='http://$loadgen_ip'
+    else
+        loadgen_ip='[not found]'
+    fi
+    GREEN='\033[0;32m'
+    COLOR_RESET='\033[0m'
+    echo -e $GREEN
+    log ""
+    log ""
+    log "--------------------------------------------------------------"
+    log "Stackdriver Sandbox deployed successfully!"
+    log ""
+    log "     Stackdriver Dashboard: https://app.google.stackdriver.com/accounts/create"
+    log "     Google Cloud Console Dashboard: $gcp_path"
+    log "     Hipstershop web app address: http://$external_ip"
+    log "     Locust load generator web interface: $loadgen_ip"
```

Comment:
Remove "locust". It's implementation detail. If we decide to switch to a different tool loadgen wrapper tool shouldn't change.